### ISSUE TITLE
Show PowerOff state in KVM screnn when host is powered off

### ISSUE THREAD

**Issue by leiyu-bytedance** [openbmc/webui-vue#74]
**Is your feature request related to a problem? Please describe.**
When a user opens the KVM page when host is powered off, the page indicates that the KVM is connected, but it shows a black screen.
Although it's expected, this somehow confuses the user who may think the KVM is not working.

**Describe the solution you'd like**
It will be much better to show some text like "Powered Off" on the black screen to tell the user the host is powered off.

**Describe alternatives you've considered**
N/A

**Additional context**
In other BMC implementations, the KVM shows additional information based on the host state, e.g. when the host is powered off, it shows "Powered Off" text; when the host is powered on but no VGA output, it shows "No Signal" text.
This helps a lot for the user to get to know the host status.

**Comment by derick-montague** [comment]
It sounds like the power status in the header was not noticed. We do have some other pages where we are showing the power status on the page as well.

When you say users are confused, does that mean you think that users will be confused or you have interviewed or worked with users that have had issues?

**Comment by leiyu-bytedance** [comment]
> It sounds like the power status in the header was not noticed. We do have some other pages where we are showing the power status on the page as well.

Yup, the power status is in the header, but the user who uses the KVM usually focuses on the KVM screen.

> When you say users are confused, does that mean you think that users will be confused or you have interviewed or worked with users that have had issues?

We have real users who already complain about this. They would like to see text like "Powered Off" on the KVM screen, instead of in the header of the web page.
